Twig/DoctrineExtension.php

```diff
@@ -37,9 +37,10 @@ class DoctrineExtension extends \Twig_Extension
     public function getFilters()
     {
         return array(
-            new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
+            new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery'), array('deprecated' => true)),
             new \Twig_SimpleFilter('doctrine_pretty_query', 'SqlFormatter::format', array('is_safe' => array('html'))),
```

Comment:
it wasn't used ... until this morning where we introduced the "View formatted query" action: https://github.com/doctrine/DoctrineBundle/pull/452/files#diff-3fa34991d339dfb273791fb46a6cff1cR186